libs/strings/storage_string/README.md

```diff
@@ -0,0 +1,58 @@
+<p align="center">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset=".docs/storage-string-logo-dark-theme.png">
+        <img alt="SwayApps logo" width="400px" src=".docs/storage-string-logo-light-theme.png">
+    </picture>
+</p>
+
+# Overview
+
+The StorageString library provides an interface to store UTF-8 encoded strings of dynamic length in Sway. The `StorageString` can be used in combination with the `String` type.
+
+The `StorageString` is stored as a collection of tightly packed bytes. This differs from Sway's built in `str` because the size cannot be known at compile time and the length is dynamic. 
```

Comment:
Could this maybe use some distinction between this type and the StorageBytes type?
As they both store a collection of tightly packed bytes it may be confusing to users. 
I think the addition of a `char` type will help here. Using the `StorageString` now sets you up for when we have better string support in the core language.

Reply:
Changed to state it stores the underlying data of the `String` type